crates/holochain/src/conductor/conductor.rs

```diff
@@ -1269,7 +1270,18 @@ mod builder {
                     .unwrap();
                 keystore
             } else {
-                spawn_lair_keystore(self.config.keystore_path.as_deref()).await?
+                let passphrase = match &self.config.passphrase_service {
+                    Some(PassphraseServiceConfig::FromConfig { passphrase }) => {
+                        tracing::warn!("USING INSECURE PASSPHRASE FROM CONFIG--This defeats the whole purpose of having a passphrase. (unfortunately, there isn't another option at the moment).");
+                        // TODO - use `new_mem_locked` when we have a secure source
+                        sodoken::BufRead::new_no_lock(passphrase.as_bytes())
+                    }
+                    oth => {
+                        panic!("We don't support this passphrase_service yet: {:?}", oth);
+                    }
```

Comment:
Is there no way to support no passphrase? Can we just set it to something arbitrary in the None case?

Reply:
I don't think we should encourage poor configuration... at least if they have to put a passphrase in a config file they should realize they're doing something insecure... For that matter, @maackle, should we rename this option to something like `danger_insecure_from_config`?

Reply:
Sure, that sounds good. And if you want to ensure that a config item is provided for this, then let's take away the `Option` so that it's required.

Reply:
Done.